Controller/AdminSecurityController.php

```diff
@@ -64,12 +64,21 @@ public function loginAction()
             return new RedirectResponse($refererUri && $refererUri != $request->getUri() ? $refererUri : $this->container->get('router')->generate('sonata_admin_dashboard'));
         }
 
+        // TODO: To be removed when the major version is changed
+        $resetRoute = 'sonata_user_admin_resetting_request';
+
+        // TODO: To be removed when the major version is changed
```

Comment:
You should add a `@trigger_error` for deprecation and specify sincer version deprecation and in which version it should be removed on both comments and triggers.